tests/client_test.py

```diff
@@ -42,16 +42,45 @@ def valid_label_properties():
 
 def score_response_json():
     return """{
-      "status":0,
+      "status": 0,
       "error_message": "OK",
-      "user_id":"12345",
+      "user_id": "12345",
       "score": 0.55
     }"""
 
 
+def action_response_json():
+    return """{
+        "actions": [
+            {
+                "action": {
+                    "id": "freds_action"
+                },
+                "entity": {
+                    "id": "Fred"
+                },
```

Comment:
Is there supposed to be something in this response that says what kind of entity this is -- i.e., user or order?